test-app/tests/unit/dom/blur-test.js

```diff
@@ -18,6 +18,10 @@ if (isEdge) {
   blurSteps = ['focusout', 'blur'];
 }
 
+if (isChrome) {
```

Comment:
Why did we not do `selectionchange` for Firefox since that supports it also?

Reply:
The behavior is slightly different.

When I was working on this PR, I had 5 browsers open
- FireFox
- FireFox Developer Edition
- Safari
- Chrome
- Chrome Canary

😅